airflow/gcp/operators/pubsub.py

```diff
@@ -403,29 +617,43 @@ class PubSubPublishOperator(BaseOperator):
         For this to work, the service account making the request
         must have domain-wide delegation enabled.
     :type delegate_to: str
+    :param project: (Deprecated) the GCP project ID where the topic will be created
+    :type project: str
     """
-    template_fields = ['project', 'topic', 'messages']
+    template_fields = ['project_id', 'topic', 'messages']
```

Comment:
```suggestion
    template_fields = ['project', 'project_id', 'topic', 'messages']
```

Reply:
> I had to revert these changes (only `"project"` shall remain) because of the errors due to errors:
> ` File "/workspace/airflow/models/baseoperator.py", line 662, in _do_render_template_fields content = getattr(parent, attr_name) AttributeError: 'PubSubPublishOperator' object has no attribute 'project'`